R/aggregate_cube.R

```diff
@@ -22,6 +22,41 @@
 
 aggregate_cube <- function(mcube, timegroup = NULL) {
 
+  # Check that mcube is a dataframe or tibble
+  if (!is.data.frame(mcube)) {
```

Comment:
stopifnot("Error: 'mcube' must be a dataframe or tibble." =
              inherits(mcube, "data.frame"))